src/App.js

```diff
@@ -1,12 +1,39 @@
+import styled from 'styled-components'
 import questions from './assets/questions.json'
 import Form from './components/Form'
+import findHouse from './services/findHouse'
 
 function App() {
-  return <Form questions={questions} onSaveHouse={saveHouse} />
+  return (
+    <PageLayout>
+      <Form questions={questions} onSaveHouse={saveHouse} />
+    </PageLayout>
+  )
 
   function saveHouse(data) {
-    console.log(data)
+    const house = findHouse(data, questions)
+    console.log(house)
   }
 }
 
+const PageLayout = styled.div`
```

Comment:
Instead of using a `div`, you could also work with a `main` element as this is more semantic or? 🤔

Reply:
Good idea!